.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -1,37 +1,49 @@
 ---
 name: Device Request
-about: This form helps in requesting a new device to be supported.
-title: "[DEVICE SUPPORT REQUEST] - Devicename"
+about: Request a new device to be supported.
+title: "Device Support: Device name"
 labels: Device Request
 assignees: ''
 
 ---
 
-Before requesting a device, please make sure to search the open and closed issues for any requests in the past. Sometimes devices have been requested before but are not implemented yet due to various reasons. If there are no hits on your Product Number, please proceed.  
+<!--
+  - Before requesting a device, please make sure to search the open and closed issues for any requests in the past.
+  - Sometimes devices have been requested before but are not implemented yet due to various reasons.
+  - If there are no hits on your for the device, please proceed.
+  - If you're unsure if the device support is already requested, please ask for advise in our Discord chat: https://discord.gg/QFhTxqN
 
-**Device Name:**
+-->
 
-**Device Type:**
-- [ ] Light
-- [ ] Lock
-- [ ] Remote
-- [ ] Sensor
-- [ ] Siren
-- [ ] Thermostat
-- [ ] Other: 
+## Device
 
-**Device Manufacturer as per deCONZ GUI basic cluster:**
+- Product name: The device name as shown on the product or package.
+- Device Type: (Light / Lock / Remote / Sensor / Siren / Thermostat)
```

Comment:
The list i provided, creates check boxes: 
**Device Type:**	## Device
- [ ] Light	
- [ ] Lock	
- [ ] Remote	
- [ ] Sensor	
- [ ] Siren	
- [ ] Thermostat	
- [ ] Other:

Reply:
Check boxes are cool, but as a maintainer who is reading the request I'm not interested in N options which don't match the device but only the one which actually represents the device. 

It's a matter of reading flow and information density. I can live with the check boxes but would prefer a linear dense bullet point list with just the essential information describing the device.

What's the opinion of the other developers here?

Reply:
How about giving the checkboxes a shot and see how it works?